### PR TITLE
Improve restore latest backup and create disaster recovery volume two bulk actions

### PR DESCRIPTION
**Pull request by a110605**
## Goal
- Allow users to configure each volume detail in modal when click `restore latest backup` and `create disaster recovery volume`
- Lock backing image field and pre-fill wth corresponding backing image name
- Some code refactor
   - rename some components name with Modal suffix
   - remove unneeded states (baseImage, bulkRestoreData, isBulkRestore) in src/models/backup.js
   - fix some typos

## Issue
[[IMPROVEMENT] Restore Latest Backup should be applied with BackingImage name value](https://github.com/longhorn/longhorn/issues/7560)

## Test Result
**Restore latest backup**

restore_latest_backup

**Create Disaster Recovery Volume**
create_DR_volume

### CODE REVIEW

File: src/components/ReactCron/ReactCron.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const steps = [
     title: 'Month',
     content: 'month',
   },
-];
```
Comment (a110605) [inline src/components/ReactCron/ReactCron.js:29]: This file is auto formatted by VScode lint plugin. Just some space and string concat correction

**Comment by a110605** [comment]
As per discussion with @ChanYiLin , we removed `encrypted` option in `Create DR Volume` & `Restore Latest Backup` modals for now. Since LH UI doesn't provide any crediential input page, the volume encypted feature will failed in backend.

**Review by torchiaf — DISMISSED** [review]
LGTM

**Review by torchiaf — APPROVED** [review]
LGTM

**Pull request merged**